mycroft/messagebus/client/client.py

```diff
@@ -14,47 +14,46 @@
 #
 import json
 import time
-from threading import Event
 import traceback
+from threading import Event
 
-from .threaded_event_emitter import ThreadedEventEmitter
-from websocket import (WebSocketApp, WebSocketConnectionClosedException,
-                       WebSocketException)
+from websocket import (
```

Comment:
Why do you prefer the shorter style?  I know this is more lines of code but IMO it is easier to read.

Reply:
My reasoning is that it's not information that is of great interest since it's not logic so it doesn't matter if it's easy or slightly less easy to read just that it doesn't distract from the reasonI'mve opened the file in the first place.